largest_sum_contiguous_subarray/LargestSumContiguousSubarray.java

```diff
@@ -0,0 +1,39 @@
+/* 
+   *   Implementation of famous dynamic programming problem
+   *   Largest Sum Contiguous Subarray
+   *   Kadane Algorithm 
+   *   Time complexity  O(n)
+*/
+package largestsum;
+
+public class LargestSumContiguousSubarray {
+
+	private static void maxSumSubArray(int[] array) {   //  maximum sum method implemention
+		int prevSum;
+		int currentSum;
+		int i;
+		prevSum = array[0];						// initialize current sum amd previous sum
+		currentSum = array[0];
+
+		for(i = 1; i < array.length; i++){
```

Comment:
Please add a space after keywords like `if`, `for`, etc.

```java
for (i = 1; i < array.length; i++) {
```
Also take care of space after closing brace of `for` and similar situations in other parts of code.